sectionsUtilityTool/README.txt

```diff
@@ -4,15 +4,23 @@
  
  a) sectionsTool$ mvn clean install
  b) copy to tomcat/webapp
- c) Add the property file on your linux box sectionsToolProps.properties. Then in JAVA_OPTS -DsectionsToolPropsPath=file:/Users/pushyami/sections/sectionsToolProps.properties 
- d) Run this as http://localhost:port/sectionsUtilityTool
+ c) Add 2 property files on linux box sectionsToolPropsSecure.properties and sectionsToolPropsLessSecure.properties Then in JAVA_OPTS add  2 -D's 
+		 1) -DsectionsToolPropsPathSecure=file:/<file-path>/sectionsToolPropsSecure.properties 
+		 2) -DsectionsToolPropsPathLessSecure=file:/<file-path>/sectionsToolPropsLessSecure.properties 
+ d) Run this as http://localhost:port/sectionsUtilityTool/?testUser=<uniquename>.
+     no need of testUser in Prod as we will enable the cosign for authentication and we will be getting remote user info from that.
  
-2) [sectionsToolProps.properties]
- Add the following 2 properties to this file. 
+2) [sectionsToolPropsSecure.properties]
+ Add the following 3 properties to this file. 
  # paste admin token here
 canvas.admin.token=
 # eg.https://umich.test.instructure.com
 canvas.url=
+# when cosign/local development is not enable set it true. Production should always be set it to false. We are enabling testUser for local develpemt and with this variable testUser will never is enabled. 
```

Comment:
Please provide a better comment indicating what this property does, rather than what environment it runs in. For example:
# If "use.test.url" is true, users will be able to execute the tool as if authenticated as the user specified in the URL parameter ?testUser=<uniqname>